fares-data-build-tool/create-data
# fix: Period Products with more than one SOP now have the additional SOP's referenced

# Description

-   made change to allow multiple sales offers packages to be referenced in the neTEx

# Testing instructions

- CD to neTEx ouput and run npm run test
- Create multiple sales offer packages, check that each one is being referenced against the product by creating the netex and ensure that all the sales offer packages selected are matched against the product

# Type of change


# Checklist:
